lochness/transfer/__init__.py

```diff
@@ -308,10 +309,10 @@ def lochness_to_lochness_transfer_s3(Lochness, general_only: bool = True):
             {source_directory}/ \
             s3://{s3_bucket_name}/{s3_phoenix_root}'
 
-    proc = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE)
-    proc.wait()
-
-    outs, _ = proc.communicate()
+    print('Executing aws s3 sync function')
+    print(re.sub('\s+', ' ', command))
```

Comment:
What in `command` are we substituting using this line?

Reply:
Replacing line breaks and multiple spaces into a single space, just for a better visibility for myself.